packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -362,6 +366,8 @@ export class TracePageImpl extends React.PureComponent<TProps, TState> {
 
     const isEmbedded = Boolean(embedded);
     const hasArchiveStorage = Boolean(storageCapabilities?.archiveStorage);
+    const isMemoryStorage = storageCapabilities?.storageType === 'memory';
```

Comment:
the UI was never made aware of the actual storage type implemented by the backend. Where is this coming from? There is no such property in storageCapabilities. But there is a flag that tells if archive storage is enabled.